dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/GitHubSourceCodeRepo.java

```diff
@@ -895,7 +905,7 @@ public void syncUserMetadataFromGitHub(User user) {
             GHMyself myself = github.getMyself();
             User.Profile profile = new User.Profile();
             profile.name = myself.getName();
-            profile.email = myself.getEmail();
+            profile.email = getEmail(myself);
```

Comment:
I wonder if to be safe, you shouldn't do `getEmail(myself) || myself.getEmail()`? Would that ever happen, that there's not a primary email, but the profile has an email?

Reply:
Is it possible to have an email thats not primary? I thought GitHub automatically sets your only email as the primary email, or the first email you listed as the primary.

Reply:
I don't know, I'm just asking. :) Can you remove all your emails from your GitHub account, or does it require you to have at least one?

Reply:
GitHub requires me to have at least one email. I am unable to remove all of my emails.